src/status_im/contexts/communities/utils.cljs

```diff
@@ -17,4 +17,5 @@
   (->> (get-in db [:wallet :accounts])
        (vals)
        (remove :watch-only?)
+       (filter :operable?)
```

Comment:
In trying to stay more consistent with the name you chose for the subscription `:wallet/operable-accounts-without-watched-accounts`, we could rename this function to `sorted-operable-and-without-watched-accounts` or something like that.

I'm wondering, doesn't _operable_ already captures the essence of an account where transactions can be made and which are not watched? I imagine a watch-only account can't be used for transactions. In that case, the name of the function could be just `sorted-operable-accounts`?

If you judge `operable` already ignores watch-only accounts, then we could shorten the subscription names too.

Reply:
Renamed here https://github.com/status-im/status-mobile/pull/20636/commits/fba717d0c90b37d623f4cdf94dc6c1f50135d227 👍  

> I'm wondering, doesn't operable already captures the essence of an account where transactions can be made and which are not watched? I imagine a watch-only account can't be used for transactions. In that case, the name of the function could be just sorted-operable-accounts?

The problem is the **_operable_** key alone is unreliable as we get the same key for watch-only accounts. It will be fully operable for a watch-only account (like the generated accounts). Additionally, we need to depend on the account's **_type_**.